Source/Plugins/ImportExport/ImportExportFilters/DxReader.cpp

```diff
@@ -243,18 +241,16 @@ void DxReader::dataCheck()
       if(!m_InStream.open(QIODevice::ReadOnly | QIODevice::Text))
       {
         QString ss = QObject::tr("Error opening input file: %1").arg(getInputFile());
-        setErrorCondition(-100);
-        notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
+        notifyErrorMessage("", ss, -100);
         return;
       }
 
       int32_t error = readHeader();
       m_InStream.close();
       if(error < 0)
       {
-        setErrorCondition(error);
         QString ss = QObject::tr("Error occurred trying to parse the dimensions from the input file. Is the input file a Dx file?");
-        notifyErrorMessage(getHumanLabel(), ss, -11000);
+        notifyErrorMessage("", ss, error);
```

Comment:
I understand how this happened, but I'm not sure how I feel about a variable error code like this.  The previous implementation set one error code and displayed another.  That sounds questionable to me from a coding / debugging perspective since it's not well defined which one is valid.  My vote is to set this back to a hard -11000 so that anyone debugging can search based on the error code.

Reply:
There are quite a few of those in there, I'll need to search for them.  I agree with that idea.